### PR TITLE
Redesign of zombie Wrestler

### PR DESCRIPTION
**Pull request by Rathner**
#### Summary
Balance "Redesign Zombie Wrestler"

#### Purpose of change
Bio_Op_Takedown is not lore friendly due to zombies not have the knowledge of how to use mechanical leverage to take someone down to the ground. Changing the wrestler to be more lore friendly while keeping it unique and interesting was the primary goal of this change.

#### Describe the solution
Removal of Bio_op_takedown. Added a second grab to fit its new description and its mechanics. Reduced its health so players have a better chance of killing it after being thrown or caught off guard. Changed its weight and size. Changed the way its dragging works. Gave it decent cooldowns for both its grabs. Increased its difficulty.

#### Describe alternatives you've considered
Removal of bio_op_takedown and leave it as is. Removal of the creature entirely. Rename/rework the bio_op_takedown. I feel like most no brainer solutions would take away from an interesting enemy type. So reworking to be equally unique seemed to be the best decision. Considered putting it in the grappler evolution line, but decided to leave it in the brute line. I might change this later based on feedback I see.

#### Testing
Tested in a new world with a default survivor and balanced it around that at different skill levels. It tends to throw you away after grabbing you, giving time to shoot it. Melee is possible against it with a default survivor, no skills and a machete but a definite gamble. Very deadly in large groups and is definitely a high priority target. I wanted to try and keep it around the same difficulty as before and I feel that it presents a similar risk as before. 

#### Additional context
There has been a hand full of complains of the bio_op_takedown ability being a legacy not lore friendly ability for something that isn't feral. Since no good solutions where being presented, I came up with my own.

### CODE REVIEW

File: data/json/effects.json [hunk 1]
```diff
@@ -340,7 +340,7 @@
     "apply_message": "You're staggered off-balance.",
     "max_duration": "5 s",
     "base_mods": { "speed_mod": [ -20 ] },
-    "removes_effects": [ "grabbing", "grabbing_2" ],
+    "removes_effects": [ "grabbing", "grabbing_2", "grabbing_3" ],
```
Comment (Maleclypse) [inline data/json/effects.json:343]:
```suggestion
    "removes_effects": [ "grabbing", "grabbing_2", "grabbing_3" ]
```

File: data/json/effects.json [hunk 2]
```diff
@@ -340,7 +340,7 @@
     "apply_message": "You're staggered off-balance.",
     "max_duration": "5 s",
     "base_mods": { "speed_mod": [ -20 ] },
-    "removes_effects": [ "grabbing", "grabbing_2" ],
+    "removes_effects": [ "grabbing", "grabbing_2", "grabbing_3" ]
```
Comment (Maleclypse) [inline data/json/effects.json:343]:
```suggestion
    "removes_effects": [ "grabbing", "grabbing_2", "grabbing_3" ],
```

**Review by Karol1223 — CHANGES_REQUESTED** [review]

File: data/json/monsters/zed_misc.json [hunk 3]
```diff
@@ -428,14 +428,14 @@
     "id": "mon_zombie_brute_grappler",
     "type": "MONSTER",
     "name": { "str": "zombie wrestler" },
-    "description": "A slab of festering muscle the size of a well-toned bodybuilder.  Seems eager to strangle the life from you.",
+    "description": "A slab of festering muscle the size of a well-toned bodybuilder. With popping and cracking sounds its arm muscles constantly lengethen and contract. It reaches out with deformed muscular hands that are the size of dinner plates.",
```
[the remark this answers is not in the document]
Comment (Kamayana) [reply data/json/monsters/zed_misc.json:431]: Ah shoot, I wrote this review yesterday but never hit submit.

[the remark this answers is not in the document]
Comment (Rathner) [reply data/json/monsters/zed_misc.json]: Fixed in Kamayanas review, thanks for the help!

[the remark this answers is not in the document]
Comment (Rathner) [reply data/json/monsters/zed_misc.json]: Fixed! Thank you.

**Pull request merged**